pkgs/top-level/all-packages.nix

```diff
@@ -13965,6 +13965,8 @@ with pkgs;
 
   ao = callPackage ../applications/graphics/ao {};
 
+  apache-directory-studio = callPackage ../applications/misc/apache-directory-studio {};
```

Comment:
It seems to belong to `applications/networking`.

Reply:
Fixed.